### ISSUE TITLE
[SSLMate] Generate representative.va.gov certificate

### ISSUE THREAD

**Issue by ph-One** [department-of-veterans-affairs/va.gov-team#99721]
## Description
_What details are necessary to understand the work that this issue is tracking?_

## Resources 
- Artifacts - i.e., issues, documentation, code, etc. - that help explain what's needed to work on this issue

## Acceptance Criteria
- [ ] _What will be created or happen as a result of this issue?_

## Refinement Guidance - Check the following before working on this issue: 
- [ ] _Epic assigned (if needed)_ 
- [ ] _Estimated (points assigned)_
- [ ] _Sprint assigned (once planned)_
- [ ] _Team member(s) assigned_
- [ ] _Understands how this work aligns with the overall platform_

**Comment by ph-One** [comment]
- Keys have been generated for dev, staging, and prod
- Self-signed certificates have been generated to allow for verification. Once verified, we can get the "correct" certificate and update param store and the TIC

**Comment by ph-One** [comment]
I need to talk with ESCC about using the self-signed cert, or if they can just pass-through to us so that we can validate our own domain. Until this validation happens, SSLMate will not provide us a signed certificate.

**Comment by ph-One** [comment]
Key is generated and domain is registered with SSLMate. Meeting with the external team next week about who needs what, and where.

**Comment by JoeTice** [comment]
Sprint 22 Update: Blocked - We did not have an opportunity to follow up this sprint with the EKS upgrade and some issues that surfaced with platform resources. Rescheduling this task for Sprint 24 as Kyle comes off of support.

**Comment by CorySohrakoffUSDS** [comment]
Hi, checking in for the accredited representative team. Has work on this been rescheduled for an upcoming sprint?

**Comment by ph-One** [comment]
@CorySohrakoffUSDS We placed other work on hold to pull this forward. I'll be working with others to see how we can get it unblocked and what the remainder of the process is.

**Comment by ph-One** [comment]
I've sent out an email [to the VA] and am waiting for a response on how exactly we can move forward with this.

**Comment by davidpetras** [comment]
PO Driven, continuing focused work rolling into Sprint 26

**Comment by ph-One** [comment]
Meeting setup for Monday to help get everyone (cross-team) in sync and to understand the needs.

**Comment by ph-One** [comment]
Working with the representative.va.gov team to get their request in and pointed at the revproxy CNAME so that SSLMate will release a certificate to us. There is nothing left to do on our part besides stay in touch and be available to the other team.
